tensor_genn/algorithms/relu_ann.py

```diff
@@ -255,7 +258,7 @@ def evaluate(self, X, y=None):
                     npop.vars["Vmem"].view[:] = random.uniform(self.Vres,self.Vthr)
                     self.g_model.push_state_to_device("if"+str(j))
                 
-            self.current_source.vars['magnitude'].view[:] = X[i] / 100.
+            self.current_source.vars['magnitude'].view[:] = X[i]*1000.
```

Comment:
Where are these 'magic numbers' coming from? As discussed by email I think you could well be 'over-saturating' the input to the neurons. Assuming that X is normalised (i.e. everything is between 0 and 1), you probably want to scale by ``($(Vthr) / (DT / $(Cm)`` so an input of 1 makes the neuron spike immediately and lower levels of input result in longer time to first spike.